docs/resources/service_compute.md

```diff
@@ -116,6 +116,7 @@ $ terraform import fastly_service_compute.demo xxxxxxxxxxxxxxxxxxxx@2
 - `product_enablement` (Block Set, Max: 1) (see [below for nested schema](#nestedblock--product_enablement))
 - `resource_link` (Block Set) A resource link represents a link between a shared resource (such as an KV Store or Config Store) and a service version. (see [below for nested schema](#nestedblock--resource_link))
 - `reuse` (Boolean) Services that are active cannot be destroyed. If set to `true` a service Terraform intends to destroy will instead be deactivated (allowing it to be reused by importing it into another Terraform project). If `false`, attempting to destroy an active service will cause an error. Default `false`
+- `stage` (Boolean) Conditionally enables new service-versions to be staged. The apply step will create a new draft version but will not stage it if this is set to `false`. Default `false`
```

Comment:
> The apply step will create a new draft version ....
nit pick; only if the latest version is locked (activated)

Reply:
This is correct as it is written; if `stage = false` then the existing behavior will be followed, which is to always create a draft version even if `cloned_version` is not locked.

The only time that the `locked` status is checked is when `stage = true`.

However... since this is causing confusion, I'll rewrite this to not incorporate multiple negative conditions.

Reply:
And now that I'm writing proper docs, I see that you are correct!

The existing behavior for `activate = true` has not changed, but the behavior for `activate = false` has changed; previously `activate = false` would result in cloning even if `cloned_version` was not locked, but now that cloning will not happen unless `cloned_version` is locked. I'm adding documentation to describe this, I don't expect that any customers would be relying on the 'extra' cloning behavior, the new behavior is more logical for when `activate = false`.